cookbook/how_to_create_and_customize_application_menu.rst

```diff
@@ -43,42 +41,50 @@ First you have to go to your default controller, create your action and specify
          */
         public function indexAction()
         {
-            return array('name' => "hello link");
+            return array('name' => "John Doe");
         }
     }
 
+.. seealso::
 
-Next you should specify action template (file path is src/Acme/Bundle/NewBundle/Resources/views/Default/index.html.twig)
-with the following content:
+    You can read more about both the ``@Route`` and the ``@Template`` annotation in the
+    `SensioFrameworkExtraBundle documentation`_.
+
+    .. _`SensioFrameworkExtraBundle documentation`: http://symfony.com/doc/current/bundles/SensioFrameworkExtraBundle/index.html
+
+Next, you should create a template for your action with the following content:
 
 .. code-block:: html+jinja
     :linenos:
 
+    {# src/Acme/Bundle/NewBundle/Resources/views/Default/index.html.twig #}
     {% extends "OroUIBundle:Default:index.html.twig" %}
     {% block content %}
-    {{ name }}!
+         <h1>Hello, {{ name }}!</h1>
     {% endblock content %}
 
-And the latest thing - you should register your bundle routes. To do that you have to create file routing.yml
-in src/Acme/Bundle/NewBundle/Resources/config/oro/routing.yml with the following content:
+And finally, you should register your bundle routes. To do that you have to create a ``routing.yml`` file
+with the following content:
 
 .. code-block:: yaml
     :linenos:
 
+    # src/Acme/Bundle/NewBundle/Resources/config/oro/routing.yml
     acme_new_bundle:
         resource:     "@AcmeNewBundle/Controller"
         type:         annotation
-        prefix:       /user
+        prefix:       /acme
 
 
-Create and fill navigation.yml
+Create navigation configuration
```

Comment:
Imo you should add a label before the headline to not break existing references to the legacy anchor like this:

```rst
.. _create-and-fill-navigation-yml:

Create navigation configuration
-------------------------------
```